pkg/pgmodel/ingestor/flags.go

```diff
@@ -0,0 +1,37 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package ingestor
+
+import (
+	"flag"
+	"time"
+
+	"github.com/timescale/promscale/pkg/pgmodel/ingestor/trace"
+)
+
+type Flags struct {
```

Comment:
This is not really in line with existing convention. We usually define a config struct that is passed into `ParseFlags` to populate configurable fields.

Reply:
You are right, the naming was off. I renamed this to Config and had the other struct renamed to Parameters. I do not want to combine the CLI-settable fields with the non-cli-settable fields. Thats why I keep two structs...

Reply:
I did recognize your idea but (1) It's not followed through the rest of the codebase (so this will be a snowflake) (2) The cut is not really clean because `NumCopiers` is configurable but it's not part of the `Config` struct (3) From the code perspective I am not really sure if it matters where the value is coming from (default or user) . Anyhow I'll approve the PR so you are not blocked on me.